.github/workflows/build.yml

```diff
@@ -124,21 +124,20 @@ jobs:
         # NOTE: Some test cases in test_certifier_framework.py need the Certifier Service
         # to be up-and-running. That needs several build-and-setup steps which are part
         # of a simple_app execution test. Execution of those cases is hence moved downstream.
-         PYTHONUNBUFFERED=TRUE pytest --capture=tee-sys -v -m "not needs_cert_service" \
+        PYTHONUNBUFFERED=TRUE pytest --capture=tee-sys -v -m "not needs_cert_service" \
              test_certifier_framework.py
 
         # Just run the basic case of this test to see classes created in generated Python module
-         PYTHONUNBUFFERED=TRUE PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python \
+        PYTHONUNBUFFERED=TRUE PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python \
              pytest --capture=tee-sys -v test_certifier_protobuf_interfaces.py \
                  -k test_certifier_pb2_basic
 
         # Re-run entire test w/o capturing output
-         PYTHONUNBUFFERED=TRUE PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python \
+        PYTHONUNBUFFERED=TRUE PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python \
              pytest -v test_certifier_protobuf_interfaces.py
 
         # Exercise SWIG-Python bindings standalone unit-tests
         PYTHONUNBUFFERED=TRUE pytest --capture=tee-sys -v test_libswigpytests.py
-        PYTHONUNBUFFERED=TRUE pytest -v test_certifier_framework.py
```

Comment:
Deleted because this was added in previous commit, when several existing pytests were commented out as they were failing.

The pytests are broken up into two sets: Those that run w/o Cert Service. And those that need Cert Service to be up. This L141 will attempt to run the latter kind, too, when Cert Service is still not up. And they will fail.

Delete this line as the latter set of tests are being run later on, downstream.